packages/flutter_tools/test/commands.shard/permeable/test_test.dart

```diff
@@ -65,6 +65,12 @@ void main() {
       return _testFile('print_user_created_ancestor_no_flag', automatedTestsDirectory, flutterTestDirectory);
     }, skip: io.Platform.isWindows); // TODO(chunhtai): Dart on Windows has trouble with unicode characters in output (#35425).
 
+    testUsingContext('report correct created widget caused the error', () async {
+      Cache.flutterRoot = '../..';
+      return _testFile('print_correct_local_widget', automatedTestsDirectory, flutterTestDirectory,
+        extraArguments: const <String>['--track-widget-creation']);
+    }, skip: io.Platform.isWindows); // TODO(chunhtai): Dart on Windows has trouble with unicode characters in output (#35425).
```

Comment:
when leaving links to bugs please use full URLs, makes it easier to find them later using grep.